### PR TITLE
Initial commit dom-api-task

### PR DESCRIPTION
**Pull request by madmaxWMFU**
### DOM-task

[Demo](https://madmaxwmfu.github.io/dom-marvel) |
[Code base](https://github.com/madmaxWMFU/dom-marvel)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/madmaxWMFU/dom-task/js/app.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            break;
+    }
+});
+
+const randomArray = type => {
+    let arrayLinks = [];
+    while (arrayLinks.length < 10) {
+        let number = Math.floor((Math.random() * 1491) + 1);
+        if (arrayLinks.indexOf(number) === -1)
+            str = `https://gateway.marvel.com/v1/public/${type}?ts=1&limit=1&offset=${number}&apikey=0cc0bbc55224cb8f34ec109ab692c2cb&hash=4e7f342316da627d9a966768f39d0702`;
```
Comment (o-msh) [inline submissions/madmaxWMFU/dom-task/js/app.js]: :eyes: `str` in a global scope.

File: submissions/madmaxWMFU/dom-task/js/app.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+const checkUrl = (arr, type) => {
+    let link;
+    arr.forEach(val => {
+        if (val.type == type)
+            link = val.url;
+    });
+    return link;
+}
+
+const templateWrap = (img, title, link, about) => {
```
Comment (o-msh) [inline submissions/madmaxWMFU/dom-task/js/app.js]: `getTemplateWrap` will be better.

File: submissions/madmaxWMFU/dom-task/js/app.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    }
+    return arrayLinks;
+}
+
+const getDataApi = arrLinks => {
+    return new Promise(resolve => {
+        Promise.all(arrLinks.map(url => fetch(url).then(resp => resp.json())))
+            .then(result => resolve(result))
+            .catch(() => {
+                console.log('oops');
```
Comment (o-msh) [inline submissions/madmaxWMFU/dom-task/js/app.js]: Do you really need `console.log` in production?

Comment (madmaxWMFU) [reply submissions/madmaxWMFU/dom-task/js/app.js]: My simple mistake and inattention!(
All done!

**Comment by madmaxWMFU** [comment]
@o-msh thanks you for your review!

**Pull request merged**